qiskit_nature/drivers/second_quantization/electronic_structure_molecule_driver.py

```diff
@@ -81,7 +81,8 @@ def driver_class_from_type(
                 raise MissingOptionalLibraryError(
                     libname=driver_type, name="ElectronicStructureDriverType"
                 )
-            driver_class.check_installed()
+            # instantiating the object will check if the driver is installed
+            _ = driver_class()
```

Comment:
This implicitly implies that:
1. all driver classes can be instantiated without any arguments
2. an instantiation of the class before we checked whether the requested method is supported

Is there a better way than this? Or are these implications fine?

Reply:
Okay continuing to read on I think these are not really of concern. Leaving this here for other reviews to check and discuss :+1:

Reply:
@mrossinek We used to have a method to check if driver was installed. I removed them favoring the Terra approach of using decorators. Since there is a decorator for the __init__ and all the driver classes that are in the list of the Molecule Driver support no arguments __init__ I just test it by instantiating. Otherwise, I can reinstate the previous `check_installed` method.

Reply:
Yep makes sense :+1: When getting into later parts of the review I started to understand that a bit better, too. I left it here for completeness sake